packages/js/browserinstance.js

```diff
@@ -6,12 +6,19 @@ const ie = require('selenium-webdriver/ie');
 const edge = require('selenium-webdriver/edge');
 const fs = require('fs');
 const path = require('path');
-const readFiles = require('read-files-promise');
+
 const Jimp = require('jimp');
 const request = require('request-promise-native');
 const utils = require('../../src/utils.js');
 const ElementFinder = require('./elementfinder.js');
 const Comparer = require('./comparer.js');
+const Tree = require('../../src/tree.js');
+const Runner = require('../../src/runner.js');
+const Reporter = require('../../src/reporter.js');
+
+let tree = new Tree();
+let runner = new Runner();
+let reporter = new Reporter(tree, runner);
```

Comment:
Why do you want to have an instance of each of these in BrowserInstance? There's just one instance of each in cli.js (one for each run of smashtest).